src/main/kotlin/com.bugsnag.android.gradle/AndroidManifestParser.kt

```diff
@@ -13,11 +13,10 @@ import java.io.IOException
 import java.io.PrintWriter
 import javax.xml.parsers.ParserConfigurationException
 
-class AndroidManifestParser {
+internal class AndroidManifestParser {
 
     private val namespace = Namespace("http://schemas.android.com/apk/res/android", "android")
 
-    @Throws(ParserConfigurationException::class, SAXException::class, IOException::class)
```

Comment:
Assumed this was leftover from java -> kotlin conversion

Reply:
Yep, this is the case.